app/SharedWebComponents/Services/ApiClient.cs

```diff
@@ -85,6 +87,8 @@ public async Task<bool> ShowLogoutButtonAsync()
     public async IAsyncEnumerable<DocumentResponse> GetDocumentsAsync(
         [EnumeratorCancellation] CancellationToken cancellationToken)
     {
+        httpClient.Timeout = Timeout.InfiniteTimeSpan;
```

Comment:
Note: in the future we will also stop upload if the user tries to leave the app after a warning.